types/materializecss__materialize/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for materializecss__materialize 1.1
+// Project: https://materializecss.github.io/materialize/
+// Definitions by:  胡玮文 <https://github.com/huww98>
+//                  Maxim Balaganskiy <https://github.com/MaximBalaganskiy>
+//                  David Moniz <https://github.com/MonizDave>
+//                  Daniel Hoenes <https://github.com/broccoliarchy>
+//                  Rick Lucassen <https://github.com/SnowyLeopard>
+//                  Mauro Mascarenhas <https://github.com/mauromascarenhas/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
```

Comment:
```suggestion
```
TS3.8 is the minimum version enforced here. If you have some specific TS versioning requirements at runtime (when DT are used), please use TS config paths versining (I assume that was just a copy over)

Reply:
Removed TS version requirement in last commit mauromascarenhas@ec93c3b.